consensus/src/experimental/mod.rs

```diff
@@ -20,5 +20,6 @@
 //                               │ Decision    │ (Asynchronously)
 //                               └─────────────┘
 
+#![allow(dead_code)]
```

Comment:
Do you need the dead code?

Reply:
this is just to prevent warnings during build